File: Food.WebApi/Models/Dish.cs
using System.ComponentModel.DataAnnotations;
using System.Text.Json.Serialization;

namespace Food.WebApi.Models
{
    public class Dish
    {
        public Dish()
        {

        }
        [Key]
        public int? Id { get; set; }
        public string Image { get; set; }
        public string Name { get; set; }
        [JsonPropertyName("isavailable")]
        public bool IsAvailable { get; set; }
        public decimal Price { get; set; }
        public int? RestaurantId { get; set; }
        public virtual Restaurant? Restaurant { get; set; }
        public ICollection<Cart>? Carts { get; set; } = new List<Cart>();
        public ICollection<Order>? Orders { get; set; } = new List<Order>();
    }
}

File: FoodToOrderAngular/src/app/models/dish.ts
export class Dish{
    id:number
    image:string
    name:string
    isavailable:boolean
    price:number

    constructor(i:number,im:string,n:string,is:boolean,p:number){
        this.id=i
        this.image=im
        this.name=n
        this.isavailable=is
        this.price=p
    }
}